soh/soh/OTRGlobals.cpp

```diff
@@ -952,8 +953,7 @@ void RunCommands(Gfx* Commands, const std::vector<std::unordered_map<Mtx*, MtxF>
         gfx_end_frame();
     }
 }
-
-// C->C++ Bridge
+    // C->C++ Bridge
```

Comment:
Seems some auto-formatting may have happened here, see if you can put this back how it was.

Reply:
Fixed.